### ISSUE TITLE
All sites updated to 7.7 uses by default the Full Screen Mode

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/gutenberg#20956]
I have noticed when updating to Gutenberg version 7.7 that all the sites automatically by default turn on FullScreen mode. I need to click the 3 dots and turn off Full Screen Mode for each client site that I enter.

Please give the user the option to turn on Full Screen mode if they decide on using it.

This can become a very tedious support issue for various client sites where I have auto updating on for themes and specific plugins. 

I also see while testing.
Deactivating the Gutenberg plugin. 
Installing the Classic Editor. 
Deactivating the Classic Editor.
Activating the Gutenberg plugin.
That the FullScreen mode turns on again. 
I do believe there are various scenarios where fullscreen mode will automatically turn on.

**Comment by paaljoachim** [comment]
I went to one of my sites and noticed that in Gutenberg version 7.8.1 that the Gutenberg layout screen by default showed full screen. The site is not updated to WP 5.4 yet. It puzzled me for I believe this issue was handled.

@jorgefilipecosta @aduth

**Comment by aduth** [comment]
@paaljoachim There are many factors which could lead to this. Full screen is the new default. But yes, if you had previously used the editor, it should not set this new default. That being said, user preferences are currently stored using browser storage, so these values can reset if you clear temporary data from your browser.

Are you seeing full screen on sites where you'd previously interacted with the editor and are certain you had not cleared your browser storage?

**Comment by paaljoachim** [comment]
I logged into a site I have not been logged into for a couple of weeks. There I noticed it went into full screen. I also checked a few other sites I have not been to for a while and made sure to update to Gutenberg plugin 7.8. There I noticed the left sidebar in place.

So I will just call the one site a gremlin in the engine...:)
And leave it at that. Thanks for the response Andrew!
If I happen to notice additional sites then I will just repost here.
